Makefile

```diff
@@ -40,8 +39,16 @@ run:
 env:
 	./scripts/bootstrap-virtualenv.sh
 
-lint:
-	${PYTHONLINT} ${PYDIRS}
+lint: listoutdated flake8diff
+
+listoutdated:
+	pip list --outdated --allow-external=cafe,cloudcafe
+
+flake8diff:
+	git diff --patch --no-prefix origin/master | flake8 --diff
```

Comment:
1. Nope. That could be added, though. (I think not having it at 105 is an improvement.)
2. Yep. That's intentional. The full lint on master is just too big to fix in one PR. (run `make flake8full` to get an idea)